src/stdx.zig

```diff
@@ -940,3 +940,12 @@ pub fn unexpected_errno(label: []const u8, err: std.posix.system.E) std.posix.Un
     }
     return error.Unexpected;
 }
+
+pub fn random_int_nonzero(comptime T: type) T {
+    const value = std.crypto.random.int(T);
+
+    // Broken CSPRNG is the likeliest explanation for zero.
+    assert(value != 0);
+
+    return value;
+}
```

Comment:
Let's maybe make it

```suggestion
pub fn unique_u128() u128 {
    const value = std.crypto.random.int(u128);

    // Broken CSPRNG is the likeliest explanation for zero.
    assert(value != 0);
    assert(value != ~0);

    return value;
}
```

?

I think `unique` captures what we want here more precisely

Reply:
And we need to prevent calling this function with `u8`, which would be wrong!